pkg/oc/bootstrap/clusteradd/cmd.go

```diff
@@ -111,7 +111,9 @@ func NewCmdAdd(name, fullName string, out, errout io.Writer) *cobra.Command {
 // Start runs the start tasks ensuring that they are executed in sequence
 func (c *ClusterAddConfig) Run() error {
 	componentsToInstall := []componentinstall.Component{}
-	installContext, err := componentinstall.NewComponentInstallContext(c.openshiftImage(), c.imageFormat(), c.BaseDir, c.ServerLogLevel)
+	dockerHelper := dockerhelper.NewHelper(c.dockerClient)
```

Comment:
I really don't want this in the context.  Can't we continue passing it through the install step?